src/app/Fake.DotNet.Cli/DotNet.fs

```diff
@@ -136,7 +136,7 @@ module DotNet =
     /// Get .NET Core SDK download uri
     /// </summary>
     let private getGenericDotNetCliInstallerUrl branch installerName =
-        sprintf "https://raw.githubusercontent.com/dotnet/cli/%s/scripts/obtain/%s" branch installerName
+        sprintf "https://raw.githubusercontent.com/dotnet/install-scripts/%s/src/%s" branch installerName
```

Comment:
I'm not really certain if this should be pulling the scripts from github or the 'distribution' urls such as https://dot.net/v1/dotnet-install.sh - but thats a bigger change which would mean dropping the configurable branch, so I just tried to keep it minimal for now

Reply:
We will be sharing guidance shortly.

This issue should provide some insight: https://github.com/dotnet/install-scripts/issues/558#issuecomment-2575550277